GOVERNANCE.md

```diff
@@ -0,0 +1,49 @@
+# Project Governance (Very Minimal Governance model)
+
+This open source project is managed by a Steering Committee composed of the maintainers of this project. Maintainers are
+defined as individuals with full commit access to the project repositories.
+
+## Steering Committee
+
+The Steering Committee will be responsible for oversight of all technical, project, approval, and policy matters for the
+project. This notably includes brand and trademark management.
+
+The Steering Committee members are listed in the MAINTAINERS.md file in the repository. New maintainers (and accordingly,
+Steering Committee members) may be added or removed as outlined in [RESPONSIBILITIES.md](./RESPONSIBILITIES.md). The
+Steering Committee will appoint a Chair responsible for organizing Steering Committee activity. If the Steering Committee
+Chair is removed from the Committee (or the Chair steps down from that role), it is the responsibility of the Steering
+Committee to appoint a new Chair.
+
+The Steering Committee may, at its discretion, add or remove members who are not maintainers.
+
+## Voting
+
+The Steering Committee will strive for all decisions to be made by consensus. While explicit agreement of the entire
+Steering Committee is preferred, it is not required for consensus. Rather, the Steering Committee will determine
+consensus based on their good faith consideration of a number of factors, including the dominant view of the Steering
+Committee and nature of support and objections. The Steering Committee will document evidence of consensus in accordance
+with these requirements. If consensus cannot be reached, the Steering Committee will make the decision by a vote.
+
+The Steering Committee Chair will call a vote with reasonable notice to the Steering Committee, setting out a discussion
+period and a separate voting period. Any discussion may be conducted in person or electronically by text, voice, or video.
+The discussion will be open to the public, with the notable exception of discussions involving embargoed security issues
+or the addition or removal of maintainers, which will be private. In any vote, each voting representative will have one
+vote. Except as specifically noted elsewhere in this document, decisions by vote require a simple majority vote of all
+voting members.
+
+## Termination of Membership
+
+A maintainer’s access (and accordingly, their position on the Steering Committee) will be removed if any of the following
+occur:
+
+* Resignation: Written notice of resignation to the Steering Committee
+* Steering Committee Vote: 3/4 affirmative vote of the Steering Committee to remove a member
+* Unreachable Member: If a member is unresponsive for more than six months, the remaining active members of the Steering
+  Committee may vote to remove the member
+
```

Comment:
Another question: do we specify any rules for ratio of maintainers per affiliation?

Reply:
And , are there any size limitations (max number of maintainers) in the steering commitee

Reply:
No and no.  Once established, the steering committee can vote to add new members, so its in their control.